platform/src/main/java/org/hillview/sketches/FreqKList.java

```diff
@@ -48,18 +51,18 @@
      * The number of times each row in the above table occurs in the original DataSet
      * (can be approximate depending on the context).
      */
-    public final HashMap<RowSnapshot, Integer> hMap;
+    public final Object2IntOpenHashMap<RowSnapshot> hMap;
```

Comment:
This is an interesting case. In general it should not make a difference optimizing sketch results, such as this one, because they are by definition small. However, this one is special, since the "filter" method is also used for a very expensive computation, so it does make sense to optimize it. This class is both a sketch result and part another sketch computation.

Reply:
I run things end-to-end from the GUI, so yes, didn't see anything fishy.